webpack.config.js

```diff
@@ -1,18 +1,28 @@
 const path = require('path');
 const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
   entry: {
     app: './src-js/app.js'
   },
   output: {
-    path: path.join(__dirname, 'jekyll/assets/js/'),
-    filename: '[name].min.js'
+    // We need the flexibility to build a js file to one place and an html file
+    // in another, which are in different places in the jekyll directory tree.
+    // We set a fairly high level path and empty default path so that we can get
+    // this flexibility.
+    path: path.join(__dirname, 'jekyll/'),
```

Comment:
I'm worried by this. Whenever I've used webpack, it was much more specific and controlled. I think it would be better to have a specific folder to house our assets and then clean up the site over time so that the js assets all live there.